extra/3-magic-8-ball.js

```diff
@@ -43,10 +45,46 @@
     Very doubtful.
 */
 
+let veryPositiveAnswers = [
+  'It is certain.',
+  'It is decidedly so.',
+  'Without a doubt.',
+  'Yes - definitely.',
+  'You may rely on it.'
+];
+
+let positiveAnswers = [
+  'As I see it, yes.',
+  'Most likely.',
+  'Outlook good.',
+  'Yes.',
+  'Signs point to yes.'
+];
+
+let negativeAnswers = [
+  'Reply hazy, try again.',
+  'Ask again later.',
+  'Better not tell you now.',
+  'Cannot predict now.',
+  'Concentrate and ask again.'
+];
+
+let veryNegativeAnswers = [
+  'Don\'t count on it.',
+  'My reply is no.',
+  'My sources say no.',
+  'Outlook not so good.',
+  'Very doubtful.'
+];
+
+let allAnswers = [...veryPositiveAnswers, ...positiveAnswers, ...negativeAnswers, ...veryNegativeAnswers];
```

Comment:
I don't think they have met spread yet. I would prefer they start with spread myself. Perhaps concat as a right answer and spread as a preferred answer?